js/plugins/drupalentity/plugin.js

```diff
@@ -200,6 +197,18 @@
   }
 
   /**
+   * Returns whether or not the given element is a editable drupalentity widget.
+   *
+   * This requires the widget to have the data-embed-button attribute present.
+   *
+   * @param {CKEDITOR.editor} editor
+   * @param {CKEDITOR.htmlParser.element} element
+   */
+  function isEditableEntityWidget (editor, element) {
+    return isEmbeddedEntityWidget(editor, element); //&& element.$.firstChild.attributes.hasOwnProperty('data-embed-button');
+  }
```

Comment:
Feels like I should be able to just remove this if the editdrupalentity command requires the data-embed-button attribute in its requiredContent.